lib/screens/app/wallet/dashboard.dart

```diff
@@ -489,4 +492,80 @@ class _DashboardState extends State<Dashboard> {
       ),
     );
   }
+
+  Widget buildSendReceiveButton() {
```

Comment:
move to its own file. 
Create a widget

Reply:
Part of the app cleanup strategy now - all new changes need to follow guidelines. 

One guideline, each widget gets its own file / class.